.eslintrc.js

```diff
@@ -52,6 +52,10 @@ module.exports = {
     // make sure we're not leveraging any deprecated APIs
     'deprecation/deprecation': 'error',
 
+    // Pending: https://github.com/typescript-eslint/typescript-eslint/issues/4820
+    // (too many false positives)
+    '@typescript-eslint/prefer-nullish-coalescing': 'off',
```

Comment:
> too many false positives

What false positives? The rule is type-aware, isn't it?
We already had it turned on as an error as well didn't we?

Reply:
...huh. I could have sworn I received many errors until I added this line. Never mind!